source/operations/monitoring/grafana.rst

```diff
@@ -0,0 +1,54 @@
+.. _minio-grafana:
+
+===================================
+Monitor a MinIO Server with Grafana 
+===================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+   
+[Grafana](https://grafana.com/) allows you to query, visualize, alert on and understand your metrics no matter where they are stored. 
```

Comment:
Markdown?

Reply:
I copied the raw code from a Github page and missed this one.